FogCreek/Glitch-Community
# Pool brownie: modularize project page styles

## Links
* Remix link: https://pool-brownie.glitch.me/

## Changes
* modularize classes defined in styles/project
* delete unused styles

## How To Test:
* visit https://pool-brownie.glitch.me/~deface-the-moon
* verify the embed and the buttons around it are styled correctly
